.github/workflows/fossa.yml

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright 2021 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: fossa
+on:
+  push:
+    branches:
+      - master
+      - release-*
+      - feature/*
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - master
+      - release-*
+      - feature/*
+  workflow_dispatch: {}
+jobs:
+  fossa-scan:
+    if: github.repository_owner == 'dapr' # FOSSA is not intended to run on forks.
+    runs-on: ubuntu-latest
+    env:
+      FOSSA_API_KEY: b88e1f4287c3108c8751bf106fb46db6 # This is a push-only token that is safe to be exposed.
```

Comment:
Seems it is yours?

Reply:
@daixiang0 it's the FOSSA token from CNCF. Same that we use for other repositories.

Reply:
I refer to making it hide and getting it from GH ENV to reduce potential risks。

Reply:
FOSSA's push-only token is safe to be used in CI directly 
https://github.com/marketplace/actions/fossa-action#push-only-api-token
https://docs.fossa.com/docs/api-reference#push-only-api-token 

Similar example with Keda: https://github.com/kedacore/keda/blob/main/.github/workflows/fossa.yml